src/core/MeganavContentPlatform/component.jsx

```diff
@@ -4,107 +4,109 @@ import T from "prop-types";
-        Examples
-      </h3>
-      <ul aria-labelledby="meganav-platform-panel-list-examples">
-        <li>
-          <a href={absUrl("/examples/avatar-stack")} className="group ui-meganav-media">
-            <p className="ui-meganav-media-heading">Avatar Stack</p>
-            <p className="ui-meganav-media-copy">See who is connected in a digital space.</p>
-          </a>
-        </li>
-        <li>
-          <a href={absUrl("/examples/emoji-reactions")} className="ui-meganav-media group">
-            <p className="ui-meganav-media-heading">Emoji Reactions</p>
-            <p className="ui-meganav-media-copy">React with an emoji to a message.</p>
-          </a>
-        </li>
-        <li>
-          <a href={absUrl("/examples/activity-feed")} className="ui-meganav-media group">
-            <p className="ui-meganav-media-heading">Activity Feed</p>
-            <p className="ui-meganav-media-copy">Display a list of user actions in realtime.</p>
-          </a>
-        </li>
-        <li>
-          <a href={absUrl("/examples/live-charts")} className="ui-meganav-media group">
-            <p className="ui-meganav-media-heading">Live Charts</p>
-            <p className="ui-meganav-media-copy">Visualise live metrics and data in a chart.</p>
-          </a>
-        </li>
-        <li>
-          <a href={absUrl("/examples/live-cursors")} className="ui-meganav-media group">
-            <p className="ui-meganav-media-heading">Live Cursors</p>
-            <p className="ui-meganav-media-copy">Track all cursors in realtime.</p>
-          </a>
-        </li>
-        <li>
-          <a href={absUrl("/examples/typing-indicator")} className="ui-meganav-media group">
-            <p className="ui-meganav-media-heading">Typing Indicator</p>
-            <p className="ui-meganav-media-copy">See when a user is typing a message.</p>
-          </a>
-        </li>
-      </ul>
-      <FeaturedLink url={absUrl("/examples")}>Explore all live examples</FeaturedLink>
-    </div>
+      <div className="col-span-full md:col-span-4 pt-24 pb-8 md:py-24 lg:py-32 px-24 sm:px-32 md:px-0">
+        <h3 className="ui-meganav-overline" id="meganav-platform-panel-list-examples">
+          Our Examples
+        </h3>
+        <ul aria-labelledby="meganav-platform-panel-list-examples">
+          <li>
+            <a href={absUrl("/examples/avatar-stack")} className="group ui-meganav-media">
+              <p className="ui-meganav-media-heading">Avatar Stack</p>
+              <p className="ui-meganav-media-copy">See who is connected in a digital space.</p>
+            </a>
+          </li>
+          <li>
+            <a href={absUrl("/examples/emoji-reactions")} className="ui-meganav-media group">
+              <p className="ui-meganav-media-heading">Emoji Reactions</p>
+              <p className="ui-meganav-media-copy">React with an emoji to a message.</p>
+            </a>
```

Comment:
just thinking is it nice to store the data in an object so we can just loop this to shorten the code?

Reply:
Hey @aralovelace, indeed, it would be nice! There is though some difficulty here in that the source of data would need to be served both for React and VW components. 

But I'm mostly avoiding refactoring here as the Meganav needs a bigger refactoring than that, so working in the background on a plan for that.